File: selection sort/selectionSort.java
import java.util.Scanner;
import java.util.ArrayList;

public class selectionSort {
    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        System.out.print("Enter size of your array: ");
        int n = scn.nextInt();
        ArrayList<Integer> arr = new ArrayList<>();
        for (int i = 1; i < n + 1; i++) {
            if (i == 1) {
                System.out.print("Enter 1st element: ");
            } else if (i == 2) {
                System.out.print("Enter 2nd element: ");
            } else if (i == 3) {
                System.out.print("Enter 3rd element: ");
            } else {
                System.out.print("Enter " + i + "th element: ");
            }
            int val = scn.nextInt();
            arr.add(val);
        }
        System.out.println("Input array: " + arr);
        for (int i = 0; i < n; i++) {
            int small = 1000000000;
            int index = 0;
            for (int j = i; j < n; j++) {
                if (small > arr.get(j)) {
                    small = arr.get(j);
                    index = j;
                }
            }
            int temp = arr.get(i);
            arr.set(i, arr.get(index));
            arr.set(index, temp);
        }
        System.out.println("Sorted array: " + arr);
    }
}

File: selection sort/selectionSort.py
n = int(input('Enter size of your array: '))
arr = []
for i in range(1, n+1):
    if i == 1:
        print("Enter 1st element: ", end='')
    elif i == 2:
        print("Enter 2nd element: ", end='')
    elif i == 3:
        print("Enter 3rd element: ", end='')
    else:
        print("Enter "+str(i)+"th element: ", end='')
    val = int(input())
    arr.append(val)
print("Input array: ", arr)
for i in range(n):
    small = 1000000000
    for j in range(i, n):
        if small > arr[j]:
            small = arr[j]
            index = j
    temp = arr[i]
    arr[i] = arr[index]
    arr[index] = temp
print("Sorted array: ", arr)
